src/components/intern-app.html

```diff
@@ -31,24 +31,44 @@
       }
     </style>
     <header>
-      <user-list></user-list>
+      <user-list user-accounts="{{userAccounts}}"></user-list>
       <create-user></create-user>
       <edit-user></edit-user>
     </header>
   </template>
 
   <script type="module">
-    
+
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-    
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const LocalUsers = new LocalStorage();
```

Comment:
since this is technically a [Storage](https://developer.mozilla.org/en-US/docs/Web/API/Storage) object, you may want to rename this to reflect that.